aws-sdk-resources/lib/aws-sdk-resources/services/s3/multipart_file_uploader.rb

```diff
@@ -158,7 +159,7 @@ def part_size(total_size, part_size, offset)
       end
 
       # @api private
-      class PartList
+      class PartQueue
```

Comment:
Only because the `MultipartObjecCopier` uses  the name `PartQueue`. I reverted it back to `PartList`.